### PR TITLE
vscode-extensions.dracula-theme.theme-dracula: init at 2.22.3

### PR DESCRIPTION
**Pull request by tbidne**
###### Motivation for this change

Add support for Visual Studio Code extension [Dracula Theme](https://marketplace.visualstudio.com/items?itemName=dracula-theme.theme-dracula)

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [x] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 114032` at d4817985 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li><a href="https://gist.github.com/24fbeb39aa59693a1a6c76a7d5e65bfb">vscode-extensions.dracula-theme.theme-dracula</a></li></li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/maintainers-missing.md">maintainers-missing</a>

Package does not have a maintainer. Consider adding yourself?

Near pkgs/misc/vscode-extensions/default.nix:193:9:
```
    |
193 |         meta = with lib; {
    |         ^
```
</li>
  </ul>
</details>

**Comment by tbidne** [comment]
> Result of `nixpkgs-review pr 114032` at [d481798](https://github.com/NixOS/nixpkgs/commit/d481798501166700305495116b60a689364289b8) run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
> 
> 1 package built:
> 1 suggestion:
> * warning: [maintainers-missing](https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/maintainers-missing.md)
>   Package does not have a maintainer. Consider adding yourself?
>   Near pkgs/misc/vscode-extensions/default.nix:193:9:
>   ```
>       |
>   193 |         meta = with lib; {
>       |         ^
>   ```

I don't mind being a maintainer, though of course that would require adding myself to `maintainer-list.nix`. Whatever is easiest :slightly_smiling_face: .

File: pkgs/misc/vscode-extensions/default.nix [hunk 1]
```diff
@@ -189,6 +189,22 @@ let
         };
       };
 
+      dracula-theme.theme-dracula = buildVscodeMarketplaceExtension {
+        meta = with lib; {
```
Comment (SuperSandro2000) [inline pkgs/misc/vscode-extensions/default.nix:193]: Please put meta last.

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Comment by SuperSandro2000** [comment]
> I don't mind being a maintainer, though of course that would require adding myself to `maintainer-list.nix`. Whatever is easiest slightly_smiling_face .

that check does not really apply to vim or vscode plugins which are very simple. You can ignore it.

File: pkgs/misc/vscode-extensions/default.nix [hunk 2]
```diff
@@ -189,6 +189,22 @@ let
...
+      dracula-theme.theme-dracula = buildVscodeMarketplaceExtension {
+        mktplcRef = {
+          name = "theme-dracula";
+          publisher = "dracula-theme";
+          version = "2.22.3";
+          sha256 = "0wni9sriin54ci8rly2s68lkfx8rj1cys6mgcizvps9sam6377w6";
+        };
+        meta = with lib; {
+          changelog = "https://marketplace.visualstudio.com/items/dracula-theme.theme-dracula/changelog";
+          description = "Official Dracula Theme. A dark theme for many editors, shells, and more.";
```
Comment (SuperSandro2000) [inline pkgs/misc/vscode-extensions/default.nix:201]:
```suggestion
          description = "Dark theme for many editors, shells, and more";
```

**Pull request merged**